### PR TITLE
Add tests for ONNX models in object_detection_demo.py

### PR DESCRIPTION
**Pull request by anzhella-pankratova**

### CODE REVIEW

**Comment by anzhella-pankratova** [comment]
Jenkins please retry a build

**Review by Wovchena — APPROVED** [review]

**Review by eizamaliev — APPROVED** [review]

File: demos/tests/args.py
Comment (IRDonch) [inline demos/tests/args.py:55]: This seems needlessly specific; I suggest that you wait until #2291 is merged and reuse the argument class that is added there.

File: demos/tests/cases.py
Comment (IRDonch) [inline demos/tests/cases.py:606]:
```suggestion
                        ModelArg('ctdet_coco_dlav0_512'),
```

Please add trailing commas elsewhere, as well.

File: demos/tests/cases.py
Comment (IRDonch) [inline demos/tests/cases.py:625]: `single_option_cases` with one argument is redundant; you can just replace it with a `TestCase`.

**Review by IRDonch — CHANGES_REQUESTED** [review]

File: demos/tests/args.py
Comment (anzhella-pankratova) [reply demos/tests/args.py:55]: Okay

File: demos/tests/cases.py
Comment (anzhella-pankratova) [reply demos/tests/cases.py]: Updated

File: demos/tests/cases.py
Comment (anzhella-pankratova) [reply demos/tests/cases.py]: Updated

**Review by IRDonch — APPROVED** [review]

**Pull request merged**
